app/controllers/v0/sign_in_controller.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 require 'sign_in/logger'
+require_relative '../openid_application_controller'
```

Comment:
Don't include `openid_application_controller` here, make sure you're not using anything from that file, we need to keep it wholly separate from Sign in Service. The two are not related